mayan/apps/documents/forms/review_forms.py

```diff
@@ -0,0 +1,48 @@
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+from ..models.reviewer import Reviewer
+from ..models.applicant import Applicant
+from ..models.review import Review
+
+
+class ReviewForm(forms.Form):
+    reviewer = forms.CharField(label='Reviewer Name', max_length=30)
+    applicant = forms.CharField(label='Applicant Name', max_length=30)
+
+    def __init__(self, *args, **kwargs):
+        self.instance = kwargs.pop('instance')
+        super().__init__(*args, **kwargs)
+
+        # Dynamically create the fields
+        metrics = self.get_metrics()
+        for metric in metrics:
+            self.fields[metric] = forms.ChoiceField(choices=[(x, x) for x in range(1, 6)])
```

Comment:
one thing to consider for this, is if the user already left a review, then by default we make the values for each metric whatever is currently in the db

Reply:
Yes, but this will require us to retrieve the review with reviewer and applicant as keys, and therefore we go back to the question of retrieving applicant user from the document